sgv2-dynamo-api/src/main/antlr4/Expression.g4

```diff
@@ -0,0 +1,74 @@
+grammar Expression;
+
+/*********************************************
+    PARSER RULES
```

Comment:
Need to add support for CONTAINS, IN, EXISTS, and SIZE operators. See https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Query.html#Query.FilterExpression and https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.OperatorsAndFunctions.html#Expressions.OperatorsAndFunctions.Syntax